src/Request/ParamConverter/DateTimeParamConverter.php

```diff
@@ -49,7 +49,9 @@ public function apply(Request $request, ParamConverter $configuration)
         if (isset($options['format'])) {
             $date = $class::createFromFormat($options['format'], $value);
 
-            if (0 < \DateTime::getLastErrors()['warning_count']) {
+            $errors = \DateTime::getLastErrors() ?: ['error_count' => 0, 'warning_count' => 0];
```

Comment:
no need for the `error_count` count since we don't use it

Reply:
No worries - was copy and pasting what Nicolas did in Symfony :) - changed now - thanks for speedy feedback.